antismash/common/secmet/features/feature.py

```diff
@@ -51,13 +51,16 @@ class Feature:
         subclass.
     """
     __slots__ = ["location", "notes", "type", "_qualifiers", "created_by_antismash",
-                 "_original_codon_start"]
+                 "_original_codon_start", "_pseudo"]
 
     def __init__(self, location: FeatureLocation, feature_type: str,
-                 created_by_antismash: bool = False) -> None:
+                 created_by_antismash: bool = False, pseudo: bool = False) -> None:
         assert isinstance(location, (FeatureLocation, CompoundLocation)), type(location)
         if location_bridges_origin(location):
             raise ValueError("Features that bridge the record origin cannot be directly created: %s" % location)
+        self._pseudo = bool(pseudo)
```

Comment:
Since you're changing the code anyway, I'd rather see this as simply set to `False`, with no constructor argument for it, since construction with a truthy value isn't allowed. This would be an improvement over the original code.